### DISCUSSION TITLE
Подключения базы данных к Яндекс.DATALENS

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#470] [ОШБК!] [answered]
После подключения БД к DATALENS происходит следующая критическая ошибка. Не всегда ( может проработать сутки ), но В какой то момент работы БД выдает ошибку подключения и помогает только перезагрузка сервера. Может быть причиной такой ошибки являться DATALENS? DATALENS работает с автообновлением данных каждые 30 сек.

**Comment by ttm-alexeypolunin** [comment]
Ошибка то какая?

**Reply by vldekov** [reply]
![-5330309200842181927_121](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/eed012d1-ed10-4bf1-8864-15e818f313f9)

**Reply by vldekov** [reply]
Есть ограничения по количеству одновременных запросов к БД, таблице?

**Reply by ttm-alexeypolunin** [reply, marked as answer]
https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/10

**Reply by vldekov** [reply]
Дополнительно есть смысл увеличить оперативную память и количество ядер процессора на сервере?

**Reply by ttm-alexeypolunin** [reply]
Ну это вы сами по нагрузке определяйте

**Reply by vldekov** [reply]
Эти параметры сервера зависят от количества одновременных пользователей БД?
